doc/stdlib/base.rst

```diff
@@ -332,56 +455,187 @@ All Objects
 
 .. function:: isdefined([object,] index | symbol)
 
-   Tests whether an assignable location is defined. The arguments can be an
-   array and index, a composite object and field name (as a symbol), or a
-   module and a symbol.
-   With a single symbol argument, tests whether a global variable with that
-   name is defined in ``current_module()``.
+   .. Docstring generated from Julia source
+   .. code-block:: julia
+       isdefined([object,] index | symbol)
+
+   Tests whether an assignable location is defined. The arguments can be an array and index, a composite object and field name (as a symbol), or a module and a symbol. With a single symbol argument, tests whether a global variable with that name is defined in ``current_module()``.
 
 .. function:: convert(T, x)
 
-   Convert ``x`` to a value of type ``T``.
+   .. Docstring generated from Julia source
+   .. code-block:: julia
+       ::
 
-   If ``T`` is an ``Integer`` type, an :exc:`InexactError` will be raised if
-   ``x`` is not representable by ``T``, for example if ``x`` is not
-   integer-valued, or is outside the range supported by ``T``.
+                  convert(T, x)
 
-   .. doctest::
+       Convert ``x`` to a value of type ``T``.
 
-      julia> convert(Int, 3.0)
-      3
+       If ``T`` is an ``Integer`` type, an :exc:`InexactError` will be raised if
+       ``x`` is not representable by ``T``, for example if ``x`` is not
+       integer-valued, or is outside the range supported by ``T``.
 
-      julia> convert(Int, 3.5)
-      ERROR: InexactError()
-       in convert at int.jl:205
+       .. doctest::
 
-   If ``T`` is a :obj:`AbstractFloat` or :obj:`Rational` type, then it will return
-   the closest value to ``x`` representable by ``T``.
+          julia> convert(Int, 3.0)
+          3
+
+          julia> convert(Int, 3.5)
+          ERROR: InexactError()
+           in convert at int.jl:205
+
+       If ``T`` is a :obj:`AbstractFloat` or :obj:`Rational` type, then it will return
+       the closest value to ``x`` representable by ``T``.
+
+       .. doctest::
+
+          julia> x = 1/3
+          0.3333333333333333
+
+          julia> convert(Float32, x)
+          0.33333334f0
+
+          julia> convert(Rational{Int32}, x)
+          1//3
+
+          julia> convert(Rational{Int64}, x)
+          6004799503160661//18014398509481984
+
+   Converts a ``UTF32String`` to ``UTF16String``
```

Comment:
Yeah we can do this as part of the method support stuff. We could probably trim this docstring down a little anyway, since it essentially says "`convert(T, x): convert x to T; return T" several times over.